wiimote.py

```diff
@@ -47,21 +51,31 @@ def __init__(
     def get_state(self):
         return self.wm.state if self.wm else None
 
-    def get_joystick_state(self, clip_values=True):
+    def get_joystick_state(self):
         """Return a tuple of the joystick state and the min/max range"""
         if not 'nunchuk' in self.get_state():
             logging.info("state: {0}".format(self.get_state()))
-            return None, None
+            return None
         else:
-            joystick_state = self.wm.state['nunchuk']['stick']
-            if clip_values:
-                joystick_state = [
-                    clip(channel, 50, 200)
-                    for channel
-                    in joystick_state
-                ]
-
-            return joystick_state, self.joystick_range
+            joystick_state_raw = self.wm.state['nunchuk']['stick']
```

Comment:
Needs simple one liner comment above the three parts to explain that it stores the raw values, the clipped raw values and the interpolated[-1,1] values.